binfmt/libelf/libelf_addrenv.c

```diff
@@ -145,6 +145,92 @@ int elf_addrenv_alloc(FAR struct elf_loadinfo_s *loadinfo, size_t textsize,
 #endif
 }
 
+/****************************************************************************
+ * Name: elf_addrenv_restore
```

Comment:
correct comment too

Reply:
Silly copy&paste error, I'll make a patch of it